packages/web-components/fast-components/src/tooltip/tooltip.styles.ts

```diff
@@ -83,6 +83,27 @@ export const tooltipStyles: (
         ${context.tagFor(AnchoredRegion)}.right .tooltip {
             margin-left: 4px;
         }
+
+        ${context.tagFor(AnchoredRegion)}.top.left .tooltip,
+        ${context.tagFor(AnchoredRegion)}.top.right .tooltip {
+            margin-bottom: 0px;
+        }
+
+        ${context.tagFor(AnchoredRegion)}.bottom.left .tooltip,
+        ${context.tagFor(AnchoredRegion)}.bottom.right .tooltip {
+            margin-top: 0px;
+        }
+
+        ${context.tagFor(AnchoredRegion)}.top.left .tooltip,
+        ${context.tagFor(AnchoredRegion)}.bottom.left .tooltip {
+            margin-right: 0px;
+        }
+
+        ${context.tagFor(AnchoredRegion)}.top.right .tooltip,
+        ${context.tagFor(AnchoredRegion)}.bottom.right .tooltip {
```

Comment:
I think we should extract the `context.tagFor(AnchoredRegion)` to the top of the function so we only have to make that call once.

Reply:
Extracted!